tests/test_datetime_parsers.py

```
@@ -21,403 +21,378 @@
+
+    # dates_in_string
+    assert dates_in_string('Time:2000-01-01 00:00:00.00') == [(0, 0.0, 'UTC')]
+    assert dates_in_string('Time[2000-01-01 00:00:00.00 tai]') == [(0, 0.0, 'TAI')]
+    assert dates_in_string('2000-01-01 00:00:00.00 Z=now') == [(0, 0.0, 'UTC')]
+    assert dates_in_string('Today is [[2000-01-01 00:00:00.00 TDB]]') == \
+        [(0, 0.0, 'TDB')]
+    assert dates_in_string('Today is [[200z-01-01 0z:00:0z.00 TDB]]') == []
+
+    # Real-world test
+    test_path = 'test_files/cpck15Dec2017.tpc'
+    with open(test_path, 'r') as f:
```

Ruff (0.15.6)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)